src/styles/blocks/nav.scss

```
@@ -0,0 +1,51 @@
+.nav {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &__list {
+    padding: 0;
+    display: flex;
+    margin: 0;
+  }
+
+  &__li {
+    margin-right: 20px;
+    position: relative;
+  }
+  &__li:last-child {
+    margin: 0;
+  }
+  &__li &__link {
+    display: flex;
+    height: 60px;
+    align-items: center;
+    text-decoration: none;
+    color: #000;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    transition: transform 400ms, color 1s;
```

There is no transform here.

```suggestion
    transition: color 1s;
```